swift/ql/lib/codeql/swift/generated/ParentChild.qll

```diff
@@ -0,0 +1,2182 @@
+// generated by codegen/codegen.py
+import codeql.swift.elements
+
+private module Impl {
+  int getMaximumChildrenIndexElement(Element e) {
+    e = e and
```

Comment:
Is this to fix a "unused" parameter warning? If so, we normally prefer writing `exists(e)` for this.

Reply:
ah, good to know! in the end I've bypassed that and other warnings by just using `none()` for `Element` (we don't want global children :slightly_smiling_face: )